end_of_life.gemspec

```diff
@@ -32,12 +32,18 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "octokit", "~> 4.0"
-  spec.add_dependency "pastel", "~> 0.8.0"
   spec.add_dependency "dry-monads", "~> 1.3"
+  spec.add_dependency "octokit", "~> 4.22"
+  spec.add_dependency "pastel", "~> 0.8.0"
   spec.add_dependency "tty-spinner", "~> 0.9.0"
   spec.add_dependency "tty-table", "~> 0.12.0"
 
+  spec.add_development_dependency "pry", "~> 0.14"
```

Comment:
I've been using `binding.irb` for a while, and it served me well. Can we remove pry for now?

Reply:
Done :)